profiler/lib.profiler/src/org/netbeans/lib/profiler/results/cpu/cct/CCTResultsFilter.java

```diff
@@ -46,7 +46,7 @@
     }
     
     public static interface EvaluatorProvider {
-        Set/*<Evaluator>*/ getEvaluators();
+        Set<Evaluator> getEvaluators();
```

Comment:
This touches the interface of the module, there are more places in the same code, that could get some generic love, but that is not "cleanup", so could you please revert this change?